apps/automation/automation-v2/src/config/index.ts

```diff
@@ -81,11 +119,19 @@ export function getConfig(env: string): Config {
 export function chainFromId(id: number): Chain {
   if (id === mainnet.id) {
     return mainnet;
-  } else if (id === berachain.id) {
-    return berachain;
   } else if (id == sepolia.id) {
     return sepolia;
   }
   else throw new Error("unsupported chain");
 }
+
+export function getMinBalanceForChain(chainId: number): BigRational {
+  if (chainId == mainnet.id) {
+    return BigRational.fromBigIntWithDecimals(parseEther("0.1"), 18n);
```

Comment:
Can just use this instead: `BigRational.fromDecimalString(s)`

Can check BigRational for other helpers, eg there's a `BigRational.fromNumer()` too